flixel/animation/FlxAnimation.hx

```diff
@@ -277,4 +277,20 @@ class FlxAnimation extends FlxBaseAnimation
 	{
 		return _frames.length;
 	}
+	
+	/**
+	 * Appends the reverse of the animation to its frames, without duplicating the original last frame.
+	 * 
+	 * If looped is true, the original first frame is also not duplicated.
+	 * 
+	 * E.g. [0,1,2,3] becomes [0,1,2,3,2,1]   (looped)     -- Frame 0 is not played twice in a row when the loop returns to the beginning.
+	 * but  [0,1,2,3] becomes [0,1,2,3,2,1,0] (non-looped) -- Animation ends looking how it started.
+	 */
+	public function pingPongifyFrames()
```

Comment:
Hm... not a huge fan of the name... thinking about it more, do we really need this if https://github.com/HaxeFlixel/flixel/issues/1888 is implemented? With that it would be as simple as:

```haxe
anim.frames = anim.frames.concat(anim.frames.reverse());
```